mongodbatlas/mongodbatlas.go

```diff
@@ -146,8 +146,12 @@ type ListOptions struct {
 type ErrorResponse struct {
 	// HTTP response that caused this error
 	Response *http.Response
-	// The error code, which is simply the HTTP status code.
-	ErrorCode int `json:"Error"`
+
+	// The error code as specified in https://docs.atlas.mongodb.com/reference/api/api-errors/
+	ErrorCode string `json:"errorCode"`
+
+	// HTTP status code.
+	HttpError int `json:"error"`
```

Comment:
note, that `Error` field wasn't possible as clashes with `Error()` method and Go forbids this